resources/js/pages/settings/bank_data.tsx

```
@@ -0,0 +1,200 @@
+                <div className="mt-6 w-full">
+                    <HeadingSmall title="GoCardless Requisitions" description="Linked bank accounts." />
+
+                    {isLoading ? (
+                        <div className="flex flex-col items-center justify-center py-12">
+                            <div className="border-foreground h-12 w-12 animate-spin rounded-full border-4 border-t-transparent"></div>
+                            <p className="text-muted-foreground mt-4">Loading...</p>
+                        </div>
+                    ) : (
+                        <div className="mt-4 grid grid-cols-1 gap-4">
+                            {requisitions.results.map((req) => (
+                                <Requisition requisition={req} setRequisitions={setRequisitions} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)